TWLight/resources/models.py

```diff
@@ -214,11 +230,12 @@ class Meta:
     excerpt_limit_percentage   = models.PositiveSmallIntegerField(blank=True, null=True, validators = [MaxValueValidator(100)],
           # Translators: In the administrator interface, this text is help text for a field where staff can optionally provide a excerpt word limit per article in terms of percentage per article.
           help_text=_("Optional excerpt limit in terms of percentage (%) of an article. Leave empty if no limit."))
-          
-    bundle = models.NullBooleanField(
-        blank=True, null=True, default=False,
-        # Translators: In the administrator interface, this text is help text for a field where staff can specify whether users can access this as part of the Bundle.
-        help_text=_("Is this partner a part of the Bundle?"))
+
+    distribution_type = models.IntegerField(choices=DISTRIBUTION_CHOICES,
+        default=EMAIL,
+        # Translators: In the administrator interface, this text is help text for a field where staff can specify which method of account distribution this partner uses.
+        help_text=_("Which distribution method does this partner use? "
```

Comment:
Seems like this should be a little more expository about how the distribution methods should work?
It might be noting that bundle is also proxied.